app/controllers/concerns/api/v1/error_handler_concern.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module ErrorHandlerConcern
```

Comment:
Naming-wise, adding the suffix `concern` is not needed. Concerns are usually prefixed with `has` or suffixed with `able` (see https://dev.to/mainstreet/how-to-name-rails-concerns-3m86). In this case, we might not even need a concern since the code is short and not shared, so it could be placed in `ApplicationController < ActionController::API`. This way all error handling, including the pagy rescue block, would be in the same file.

Reply:
Fixed in 8ed84408013de8ec98bb335c5f3f7661cf297acc